README.md

```diff
@@ -31,18 +31,17 @@ $ docker build . -t example-web
 ```
 
 ## Try it
+### Register a Service
 
-This example can be run locally to test the flows.
+Register a Service and a Consent Template in Truid to obtain OAuth2 client credentials. The service need the following property:
```

Comment:
Should we expose the term `Consent Template` externally at this stage?

Reply:
no, let's remove it